Formula/graphviz.rb

```diff
@@ -71,6 +71,7 @@ def install
       --prefix=#{prefix}
       --without-qt
       --with-quartz
+      --enable-php=no
```

Comment:
does `--disable-php` work instead?

Reply:
Ther is no different `--enable-php=no` and `--disable-php`.
```
==> ./configure --disable-debug --disable-dependency-tracking --prefix=/usr/local/Cellar/graphviz/2.40.1 --without-qt --with-quartz --enable-php=no --with-gts --without-rsvg
```
```
----------------------------------------------------------------

graphviz-2.40.1 will be compiled with the following:

options:
  cgraph:        Yes (always enabled)
  digcola:       Yes
  expat:         Yes
  fontconfig:    Yes
  freetype:      Yes
  glut:          Yes
  ann:           No (ANN library not available)
  gts:           Yes
  ipsepcola:     No (disabled by default - C++ portability issues)
  ltdl:          Yes
  ortho:         Yes
  sfdp:          Yes
  shared:        Yes
  static:        No (disabled by default)
  qt:            No (disabled)
  x:             Yes

commands:
  dot:           Yes (always enabled)
  neato:         Yes (always enabled)
  fdp:           Yes (always enabled)
  circo:         Yes (always enabled)
  twopi:         Yes (always enabled)
  gvpr:          Yes (always enabled)
  gvmap:         Yes (always enabled)
  lefty:         Yes
  smyrna:        No (disabled by default - experimental)
  gvedit:        No (disabled)

plugin libraries:
  dot_layout:    Yes (always enabled)
  neato_layout:  Yes (always enabled)
  core:          Yes (always enabled)
  devil:         No (missing library)
  gd:            Yes
  gdiplus:       No (disabled by default - Windows only)
  gdk:           
  gdk_pixbuf:    No (gdk_pixbuf library not available)
  ghostscript:   Yes
  glitz:         No (disabled by default - incomplete)
  gtk:           No (gtk library not available)
  lasi:          No (lasi library not available)
  ming:          No (disabled by default - incomplete)
  pangocairo:    Yes
  poppler:       No (poppler library not available)
  quartz:        Yes
  rsvg:          No (disabled)
  visio:         No (disabled by default - experimental)
  webp:          No (disabled by default - experimental)
  xlib:          Yes

language extensions:
  gv_sharp:      No (mcs not available)
  gv_go:         No (disabled by default - experimental)
  gv_guile:      No (guile not available)
  gv_io:         No (disabled by default - no swig support yet)
  gv_java:       Yes
  gv_lua:        No (lua not available)
  gv_ocaml:      No (ocaml not available)
  gv_perl:       Yes
  gv_php:        No (disabled)
  gv_python:     Yes
  gv_python23:   No (disabled by default - for multiversion installs)
  gv_python24:   No (disabled by default - for multiversion installs)
  gv_python25:   No (disabled by default - for multiversion installs)
  gv_python26:   No (disabled by default - for multiversion installs)
  gv_python27:   No (disabled by default - for multiversion installs)
  gv_python34:   No (disabled by default - for multiversion installs)
  gv_R:          No (R not available)
  gv_ruby:       Yes
  gv_tcl:        No (tcl not available)

  tcldot:        No (tcl not available)
  tclpathplan:   No (tcl not available)
  gdtclft:       No (tcl not available)

Testing utilities:
  criterion:     No (Criterion unit testing framework not installed)
```

```
==> ./configure --disable-debug --disable-dependency-tracking --prefix=/usr/local/Cellar/graphviz/2.40.1 --without-qt --with-quartz --disable-php --with-gts
```
```
----------------------------------------------------------------

graphviz-2.40.1 will be compiled with the following:

options:
  cgraph:        Yes (always enabled)
  digcola:       Yes
  expat:         Yes
  fontconfig:    Yes
  freetype:      Yes
  glut:          Yes
  ann:           No (ANN library not available)
  gts:           Yes
  ipsepcola:     No (disabled by default - C++ portability issues)
  ltdl:          Yes
  ortho:         Yes
  sfdp:          Yes
  shared:        Yes
  static:        No (disabled by default)
  qt:            No (disabled)
  x:             Yes

commands:
  dot:           Yes (always enabled)
  neato:         Yes (always enabled)
  fdp:           Yes (always enabled)
  circo:         Yes (always enabled)
  twopi:         Yes (always enabled)
  gvpr:          Yes (always enabled)
  gvmap:         Yes (always enabled)
  lefty:         Yes
  smyrna:        No (disabled by default - experimental)
  gvedit:        No (disabled)

plugin libraries:
  dot_layout:    Yes (always enabled)
  neato_layout:  Yes (always enabled)
  core:          Yes (always enabled)
  devil:         No (missing library)
  gd:            Yes
  gdiplus:       No (disabled by default - Windows only)
  gdk:           
  gdk_pixbuf:    Yes
  ghostscript:   Yes
  glitz:         No (disabled by default - incomplete)
  gtk:           No (gtk library not available)
  lasi:          No (lasi library not available)
  ming:          No (disabled by default - incomplete)
  pangocairo:    Yes
  poppler:       No (poppler library not available)
  quartz:        Yes
  rsvg:          Yes
  visio:         No (disabled by default - experimental)
  webp:          No (disabled by default - experimental)
  xlib:          Yes

language extensions:
  gv_sharp:      No (mcs not available)
  gv_go:         No (disabled by default - experimental)
  gv_guile:      No (guile not available)
  gv_io:         No (disabled by default - no swig support yet)
  gv_java:       Yes
  gv_lua:        No (lua not available)
  gv_ocaml:      No (ocaml not available)
  gv_perl:       Yes
  gv_php:        No (disabled)
  gv_python:     Yes
  gv_python23:   No (disabled by default - for multiversion installs)
  gv_python24:   No (disabled by default - for multiversion installs)
  gv_python25:   No (disabled by default - for multiversion installs)
  gv_python26:   No (disabled by default - for multiversion installs)
  gv_python27:   No (disabled by default - for multiversion installs)
  gv_python34:   No (disabled by default - for multiversion installs)
  gv_R:          No (R not available)
  gv_ruby:       Yes
  gv_tcl:        No (tcl not available)

  tcldot:        No (tcl not available)
  tclpathplan:   No (tcl not available)
  gdtclft:       No (tcl not available)

Testing utilities:
  criterion:     No (Criterion unit testing framework not installed)
```